src/Domain/Insights/InsightFactory.php

```diff
@@ -167,6 +185,9 @@ private function getSniffCollector(array $config): ErrorAndDiffCollector
         $ecsContainer = EcsContainer::make();
 
         $ecsContainer->set(Configuration::class, $configuration);
+        /** @var EasyCodingStandardStyle $style */
+        $style = $ecsContainer->get(EasyCodingStandardStyle::class);
+        (new Reflection($style))->set('output', $consoleOutput);
```

Comment:
:sweat_smile: From the beginning of the PR, I wondering why we passed the $consoleOutput to the insightCollectionFactory. Now I know :+1:

Reply:
Haha yeah, It was kinda tricky passing it all the way down :D

Reply:
@olivernybroe @Jibbarth Maybe we can do better where, the domain layer should not have application logic on it.

Maybe on the service injectors layer we can do this job?

Reply:
PS: I understand that the code is kind of messy here. 😄